helix-term/src/ui/picker.rs

```diff
@@ -505,15 +505,15 @@ impl<T: 'static> Component for Picker<T> {
         let selected = cx.editor.theme.get("ui.text.focus");
 
         let rows = inner.height;
-        let offset = self.cursor / (rows as usize) * (rows as usize);
+        let offset = self.cursor / std::cmp::max(1, rows as usize) * (rows as usize);
```

Comment:
```suggestion
        let offset = self.cursor / std::cmp::max((1, rows as usize) * (rows as usize));
```

If rows is zero, `1 * 0` would still end up being a division by zero.

Reply:
I think that suggestion is wrong, it should be `std::cmp::max(1, (rows as usize) * (rows as usize))`